src/codesearch.cc

```diff
@@ -439,10 +444,24 @@ code_searcher::~code_searcher() {
     }
 }
 
+void code_searcher::chunk_files() {
```

Comment:
`chunk_files` is a super confusing name for this, since we also have a data structure called `chunk_file` :)